packages/core/src/browser/view-container.ts

```diff
@@ -699,6 +712,11 @@ export class ViewContainerPart extends BaseWidget {
         this.wrapped.title.changed.connect(fireTitleChanged);
         this.toDispose.push(Disposable.create(() => this.wrapped.title.changed.disconnect(fireTitleChanged)));
 
+        if (DescriptionWidget.is(this.wrapped)) {
+            const fireSubHeadingChanged = () => this.onDidSubHeadingChangedEmitter.fire(undefined);
+            this.toDispose.push((this.wrapped as DescriptionWidget)?.onDidDescriptionChange.event(fireSubHeadingChanged));
```

Comment:
Since you've performed a type check already, you shouldn't need the `as` cast or the `?.` conditional accessor.

Why the change in terminology here from `onDidChangeDescription` to `onDidChangeSubHeading`? I think I'd prefer that we use the `description`-type names throughout and leave the fact that they're rendered as a subheading as an implementation detail.